MessageViewController/MessageView.swift

```diff
@@ -27,6 +27,11 @@ public final class MessageView: UIView, MessageTextViewListener {
         case right
     }
 
+    internal var buttonAction: Selector?
+    internal var keyboardHeight: CGFloat = 0
```

Comment:
Are these used?

Reply:
Whoops, forgot to delete. Thanks for the catch.